cds_ils/eitems/serializers/custom_fields.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""EItem custom serializer functions."""
+
+from flask import current_app
+
+
+def field_urls(metadata):
```

Comment:
nit: naming police 🚔 
can we name the function with something suggesting that it actually transforms the login required urls? f.e. `format_login_required_urls` ?

Reply:
Done!